### PR TITLE
Links

### PR DESCRIPTION
**Pull request by HeidiHansen**

### CODE REVIEW

File: src/data/links.yaml [hunk 1]
```diff
@@ -1750,3 +1750,39 @@ links:
       Wish you could avoid airport queues, tedious transfers and long drives?
       Want to travel sustainably and enjoyably by train instead - to fantastic accommodation in the best ski resorts in the Alps?
       Snowcarbon is an independent guide founded by ski journalist Daniel Elkan to help you do exactly that.
+
+  - title: Runamics Running & Sports Gear
+    url: https://www.runamics.com
+    countries: []
+    categories: [clothing]
+    tags: [clothing]
+    description: Crowd-funded for-purpose eco-friendly alternatives to traditional sports gear.  We combine merino wool with wood-based cellulosic fibres like Lyocell. We make experiments with highly innovative fibres like biodegradable elastic fibres or “polyester-like” substitutes.  We're a climate neutral company- We avoid emissions where we can and where they can't be avoided, we compensate them. We will repair your runamics garment and send it back to you for free or discount a replacement. We use plastic-free packaging and ship globally with DHL GoGreen.
```
Comment (philsturgeon) [inline src/data/links.yaml]: Can you use the multiline YAML syntax like the above links?

```suggestion
    description: >
      Crowd-funded for-purpose eco-friendly alternatives to traditional sports gear. 
      We combine merino wool with wood-based cellulosic fibres like Lyocell. We make 
      experiments with highly innovative fibres like biodegradable elastic fibres or “polyester-like”
      substitutes.  WE're climate neutral always working to reduce our footprint, use plastic-free packaging and ship globally with DHL GoGreen.
```

Also GO GREEN WHAT IS THIS :D

File: src/data/links.yaml [hunk 2]
```diff
@@ -1750,3 +1750,39 @@ links:
...
+
+  - title: Lost in Samsara - Bags, Jewelry, and Accessories
+    url: https://www.lostinsamsara.co.uk
+    countries: []
+    categories: [clothing]
+    tags: [clothing, second-hand, jewelry]
+    description: Stylish products produced ethically from upcycled materials.  Bags made from cement bags, tires, and jewelry made from upcycled bullet casings.
+
+  - title: A Good Company - sustainable everyday products
```
Comment (philsturgeon) [inline src/data/links.yaml]: Just the title in the title then slogans can go in description.

File: src/data/links.yaml [hunk 3]
```diff
@@ -1750,3 +1750,39 @@ links:
...
+    url: https://loopalife.com/
+    featured: true
+    countries: [nl]
+    categories: [clothing]
+    tags: [clothing, second-hand]
+    description: This circular textile award winning brand sets the new standard in the textile industry by creating gorgeous quality clothing from local raw salvaged materials, recycling them without any chemicals, generating no wastewater, recycling their own products at end-of-life, researching and developing new ways to develop circular products.  They also expand their impact by organizing local sorting campaigns, workshops, collection campaigns, and facilitate other fashion companies in their circular transition.  When compared to traditional manufacturing, each sweater saves 500 liters of water, 60mJ of energy, uses zero chemicals because they sort the fibers based on color instead of bleaching and dyeing, and save at least 6kg of C02 emissions on average because everything is sourced, processed, produced, and sold locally.  
+
+  - title: Lost in Samsara - Bags, Jewelry, and Accessories
+    url: https://www.lostinsamsara.co.uk
+    countries: []
```
Comment (philsturgeon) [inline src/data/links.yaml]:
```suggestion
```

File: src/data/links.yaml [hunk 4]
```diff
@@ -1750,3 +1750,39 @@ links:
...
+      
+  - title: MakeSense Jobs
+    url: https://jobs.makesense.org
+    countries: []
+    categories: [volunteering, jobs]
+    tags: [career, job, volunteering, internship]
+    description: Makesense inspires and empowers citizens, entrepreneurs and organizations to build an inclusive and sustainable society together.  MakeSense's Job and assignment announcements platform for associations and social entrepreneurs who recruit profiles passionate about the impact. Job types include internship, freelance, civic service, partner, program, volunteer mission, mentoring.
+
+  - title: Loop A Life 100% circular fashion - truly circular and wasteless wear
```
Comment (philsturgeon) [inline src/data/links.yaml]:
```suggestion
  - title: Loop A Life - 100% Circular Fashion
```

This might also be too much, but its better.

Comment (mbifulco) [reply src/data/links.yaml]: Is it worth making a case for optional subtitles? There's plenty of links on the site whose product/company names aren't super straightforward.  What if we added a ~50char human-readable description of what the thing is?

**Comment by philsturgeon** [comment]
Yeah let’s do it!

File: src/data/links.yaml (hunk 1 above)
Comment (HeidiHansen) [reply src/data/links.yaml]: [DHL GoGreen](https://www.dpdhl.com/en/responsibility/environment-and-solutions/gogreen-program.html) is news to me.  The link outlines their sustainability goals in impressive detail.  Would be good to add.

**Pull request merged**